include/cuco/detail/pair/pair.inl

```diff
@@ -49,3 +49,11 @@ __host__ __device__ constexpr bool operator==(cuco::pair<T1, T2> const& lhs,
 }
 
 }  // namespace cuco
+
+namespace thrust {
+#include <cuco/detail/pair/tuple_helpers.inl>
+}  // namespace thrust
+
+namespace cuda::std {
+#include <cuco/detail/pair/tuple_helpers.inl>
+}  // namespace cuda::std
```

Comment:
I think the namespaces should be in the headers

Reply:
Oh, I see, this is your trick to reuse the same specializations for Thrust and libcu++.